eoscostarica/evodex
# bufix: font and text issues

## Resolve font and text issues

### What does this PR do?
- Resolve #52
- Resolve #53
- Resolve #55 
- Resolve #56  

### How should this be tested?
- yarn
- yarn start
- go to http://localhost:3000/evodex/fee
- check font and text for menu, title and subtitles 

**Note:** 
 Also you can use https://deploy-preview-58--evodex.netlify.app for testing
